Gems/Maestro/Code/Source/Cinematics/AnimSequence.cpp

```diff
@@ -377,7 +377,7 @@ IAnimNode* CAnimSequence::CreateNode(XmlNodeRef node)
 
     // Make sure de-serializing this node didn't just create an id conflict. This can happen sometimes
     // when copy/pasting nodes from a different sequence to this one. 
-    for (auto curNode : m_nodes)
+    for (const auto & curNode : m_nodes)
```

Comment:
The ampersand should be attached to the `auto`
e.g `const auto& curNode : m_nodes`